go-controller/pkg/ovn/master.go

```diff
@@ -670,7 +671,11 @@ func (oc *Controller) syncGatewayLogicalNetwork(node *kapi.Node, l3GatewayConfig
 		if err != nil {
 			return err
 		}
-		if err := addPolicyBasedRoutes(node.Name, hostIfAddr.IP.String(), l3GatewayConfigIP); err != nil {
+		relevantHostIPs, err := util.MatchAllIPStringFamily(utilnet.IsIPv6(hostIfAddr.IP), hostAddrs.List())
```

Comment:
This isn't related to your patch, but `hostIfAddr` isn't the management port's ifAddr as retrieved by `util.GetNodeManagementIfAddr`, right? Feels like we should call that `mgmtIfAddr` or similar

Reply:
Actually yes, it is the management port IP, that's what the `addPolicyBasedRoutes` function signature denotes

Reply:
yeah, naming is not good, but not going to address that here